sdk/src/main/java/com/meniga/sdk/models/sync/operators/MenigaSyncOperationsImp.java

```diff
@@ -5,18 +5,25 @@
 import com.meniga.sdk.models.sync.MenigaSync;
 import com.meniga.sdk.models.sync.MenigaSyncStatus;
 import com.meniga.sdk.webservices.requests.GetSync;
+import com.meniga.sdk.webservices.requests.StartRealmSync;
 import com.meniga.sdk.webservices.requests.StartSync;
 
 /**
  * Copyright 2017 Meniga Iceland Inc.
  */
 public final class MenigaSyncOperationsImp implements MenigaSyncOperations {
 	@Override
-	public Result<MenigaSync> startSync(long timeout) {
-		StartSync req = new StartSync();
-		req.waitForCompleteMilliseconds = timeout;
-
-		return MenigaSDK.executor().startSync(req);
+	public Result<MenigaSync> startSync(Long realmUserId, long timeout) {
```

Comment:
It's poorly set up in the api in my judgement because what it actually means is this: sync = sync all realms, sync with realm user is = only sync the realm of this user ie only sync one specific realm. It's always going to be misleading to require a realm user id when what is being done is syncing the realm - but it should probably be called syncRealm in that case.

Reply:
What do you think about two methods named like this (in this class and in `MenigaSync.java`):

```
public Result<MenigaSync> syncRealms(long timeout) { ... }
public Result<MenigaSync> syncRealm(long realmUserId, long timeout) { ... }
```

This way we say exactly what you said that we have an all realm sync and a realm sync for a userId.

Reply:
Excellent - deprecating the other ones